options.c

```diff
@@ -989,28 +989,38 @@ options_from_string_flag(struct options *oo, const char *name,
 	return (0);
 }
 
+int
+string_choice_from_options(const struct options_table_entry *oe,
```

Comment:
ℹ️ Did some refactoring and extracted part the code from `options_from_string_choice` into this separate method, so it could be used in `cmd-display-menu.c` to convert a `-b` flag string argument to the proper popup-border-lines `int`.